tests/trello/gui/resources/components/ShownMenu.resource

```diff
@@ -42,4 +47,30 @@ Update Description of Board
     Wait Until Element Is Visible           ${SAVE_DESC_BUTTON}
     Click Element           ${SAVE_DESC_BUTTON}
     Click Element           ${GO_BACK_BUTTON}
-    Click Element           ${CLOSE_SHOW_MENU}
\ No newline at end of file
+    Click Element           ${CLOSE_SHOW_MENU}
+Enter to the list archive
+    [Documentation]    accesing to the archive menu in the sidebar
+    Wait Until Element Is Enabled    ${MORE_OPTIONS_SIDEBAR}    timeout=10s
+    Click Link    ${MORE_OPTIONS_SIDEBAR}
+    Wait Until Element Is Enabled    ${ARCHIVED_ITEMS_BUTTON}            timeout=10s
+    Click Link    ${ARCHIVED_ITEMS_BUTTON}
+    Wait Until Element Is Enabled    ${SWITCH_TO_LIST_BUTTON}        timeout=10s
+    Click Link   ${SWITCH_TO_LIST_BUTTON}
+
+Close archive tab
+    [Documentation]   close sidebar tab
+    Wait Until Element Is Enabled   ${CLOSE_SHOW_MENU}            timeout=10s
+    Click Link    ${CLOSE_SHOW_MENU}
+
+Click watch/unwatch button
+    [Documentation]    click to watch/unwatch the last list on ui
```

Comment:
If it only works for a specific list, it should be expressed in the name of the keyword